classGenerator/generator/transformer.ts

```diff
@@ -0,0 +1,111 @@
+import * as ts from "typescript";
```

Comment:
Can we use package for parser and transformer?

Reply:
* I updated the `generator.ts, paser.ts ` files by only importing the relevant functions/objects from the typescript package 😀 
* In the` transformer.ts` file, we are using the following functions/ objects from the typescript package. Do we need to import them separately?
```
Node, TransformationContext,  SourceFile,  EmitHint, ScriptTarget, Printer,
NewLineKind, createPrinter, createSourceFile, updateIdentifier, updateClassDeclaration,
createNodeArray, createIdentifier, visitEachChild, visitNode, transform
```